advisors/spring-ai-advisors-vector-store/src/main/java/org/springframework/ai/chat/client/advisor/vectorstore/QuestionAnswerAdvisor.java

```diff
@@ -303,18 +192,13 @@ public Builder searchRequest(SearchRequest searchRequest) {
 			return this;
 		}
 
-		/**
-		 * @deprecated in favour of {@link #promptTemplate(PromptTemplate)}
-		 */
-		@Deprecated
-		public Builder userTextAdvise(String userTextAdvise) {
-			Assert.hasText(userTextAdvise, "The userTextAdvise must not be empty!");
-			this.promptTemplate = PromptTemplate.builder().template(userTextAdvise).build();
+		public Builder protectFromBlocking(boolean protectFromBlocking) {
```

Comment:
This is superseded by the new Scheduler input argument. I kept it for backward compatibility. And I haven't marked it as deprecated since we're trying to avoid that for RC1. But this is a candidate to be eventually removed in future releases.